### ISSUE TITLE
Salt and Fresh Water recipes not returning bucket

### ISSUE THREAD

**Issue by GiovanniPorta** [whatthedrunk/allthemods#409]
**Issue Description:**
The HarvestCraft recipes for salt and fresh water do not return the water bucket used.

**What happens:**
The bucket is eaten by the recipe and does not get returned.

**What you expected to happen:**
An empty bucket is supposed to be left in the crafting grid after the water is consumed from the water bucket.

**Steps to reproduce:**
Attempt to craft fresh water using a water bucket, or salt using a water bucket and pot.

**The Version of All the Mods:** 1.60b

**Added mods (Optifine, Fastcraft, etc.):**
None.

Same issue as posted on Reddit 20 days ago: https://www.reddit.com/r/feedthebeast/comments/5p7h5l/all_recipes_that_are_supposed_to_return_a_water/

![Recipe Broken GIF](https://cloud.githubusercontent.com/assets/14063450/22847449/c66d5016-efa1-11e6-836c-6b9ac13d4c8c.gif)

I tried adding the following to a script to fix the recipe, but while it works in a TiC crafting station, the vanilla crafting table and inventory crafting grid show a red 0 next to the bucket and then it disappears when trying to click it.
```
recipes.remove(<harvestcraft:freshwaterItem>);
recipes.addShapeless(<harvestcraft:freshwaterItem>, [<minecraft:water_bucket>.transformReplace(<minecraft:bucket>)]);
```

Example:
![Attempt to fix GIF](https://cloud.githubusercontent.com/assets/14063450/22847363/5816d13c-efa1-11e6-8264-0cf96c42ea6d.gif)

**Comment by GiovanniPorta** [comment]
I have narrowed the issue down to the Calculator mod. SonarSonic/Calculator#215

**Comment by Shinisuryu** [comment]
Thank you thank you thank you for doing the work on narrowing down the mod causing that. I had no idea why it was happening and searching the pam's issue page didn't have anything about it.

**Comment by whatthedrunk** [comment]
I finally got around to just removing Calculator in 2.0 going forward.

**Comment by GiovanniPorta** [comment]
For future reference, this got fixed in Calculator 3.1.9
